doc/glossary.rst

```diff
@@ -373,6 +373,28 @@
                 provides a gateway to both the Amazon S3 RESTful API and the
                 OpenStack Swift API. 
 
+        scrubbing
+
+                The process by which Ceph ensures data integrity. During the
+                process of scrubbing, Ceph generates a catalog of all objects
+                in a placement group, then ensures that none of the objects are
+                missing or mismatched by comparing each primary object against
```

Comment:
nit: this describes deep scrubs specifically

Reply:
Suggest moving text starting with `During` below under deep scrubbing and consolidating with the existing text.

Also, suggest refactoring this entry as `scrubs` instead of `scrubbing`.

Reply:
I want to hear more of the case for renaming the entry the plural noun "scrubs" instead of the admittedly more abstract gerund "scrubbing". In my head, "scrubbing" seems the more natural choice. I am open to the argument to the contrary, but I don't see it on first blush.